compiler/rustc_builtin_macros/src/format.rs

```diff
@@ -349,10 +482,25 @@ impl<'a, 'b> Context<'a, 'b> {
         }
     }
 
-    fn verify_count(&mut self, c: parse::Count<'_>) {
+    fn verify_count(
+        &mut self,
+        c: parse::Count<'_>,
+        inner_span: &Option<rustc_parse_format::InnerSpan>,
+        named_arg_type: PositionalNamedArgType,
+    ) {
         match c {
             parse::CountImplied | parse::CountIs(..) => {}
             parse::CountIsParam(i) => {
+                let start_of_named_args = self.args.len() - self.names.len();
+                if i >= start_of_named_args {
+                    self.unused_names_lint.maybe_push(
+                        i,
+                        named_arg_type,
+                        self.curpiece,
+                        inner_span.clone(),
```

Comment:
Is `inner_span` not `Copy`? If not, then it probably can be.

```suggestion
                        *inner_span,
```